lib/index.js

```diff
@@ -93,9 +104,17 @@ exports.runOperation = async ({operation, storage, moduleManager}) => {
     throw error;
   }
   if(isGenerateKeyOp) {
+    let suiteContextUrl = symmetric.get(invocationTarget.type);
+    if(!suiteContextUrl) {
+      ({SUITE_CONTEXT: suiteContextUrl} =
+        cryptoLd.suites.get(invocationTarget.type) || {});
+      if(!suiteContextUrl) {
+        throw new Error(`Unknown key type: "${invocationTarget.type}".`);
+      }
+    }
```

Comment:
We also need to check that the suite context is in the message, right? The json-schema we have will make sure at least *one* of the possible key suite contexts we support will be present, however, it won't check to make sure that the one we need that *matches* `invocationTarget.type` is present. For example, someone may pass a key gen op requesting to create an ed25519 key -- but use the aes keywrap context. We should also have a test to make sure this won't be accepted.

Reply:
Done here ddbb378